modules/apps/document-library/document-library-video/src/main/resources/META-INF/resources/js/components/DLVideoExternalShortcutInput.tsx

```diff
@@ -17,16 +24,20 @@ const DLVideoExternalShortcutPreview = ({labelTooltip, onChange, url = ''}) => {
 				{Liferay.Language.get('video-url')}
 
 				{labelTooltip && (
-					<ClayIcon
-						className="ml-1 text-secondary"
-						symbol="question-circle-full"
-						title={labelTooltip}
-					/>
+					<span title={labelTooltip}>
```

Comment:
To avoid adding the `span` element, I think we should do something like this:
```
<ClayIcon
    className="lfr-portal-tooltip ml-1 text-secondary"
    data-title={labelTooltip}
    role="tooltip"
    symbol="question-circle-full"
    tabIndex={0}
/>
```
We can add `role="tooltip"` since it’s a help icon, and a `tabindex="0"` to make the element focusable. Another thing: I’d take it out of the label, so when you click on the icon it doesn’t also trigger the label click.